code/drasil-printers/lib/Language/Drasil/Printing/AST.hs

```diff
@@ -50,7 +50,8 @@ infixr 5 :+:
 
 -- | Redefine the 'Sentence' type from Language.Drasil to be more suitable to printing.
 data Spec = E Expr                   -- ^ Holds an expression.
-          | S String                 -- ^ Holds a String.
+          | S String                 -- ^ Holds a string.
+          | Ch SentenceStyle TermCapitalization UID                 
```

Comment:
Missing a comment to say what `Ch` is for? Is it actually `Chunk`? That would not be quite the right way to do that -- too brute force.